Dockerfile

```diff
@@ -4,9 +4,17 @@ MAINTAINER Code Climate <hello@codeclimate.com>
 WORKDIR /usr/src/app
 COPY package.json /usr/src/app/
 
-RUN apk --update add git && \
+RUN apk --update add git jq && \
     npm install && \
-    apk del --purge git
+    git clone https://github.com/eslint/eslint.git && \
```

Comment:
In particular, it is automatically pulling in the docs associated with the version of ESLint that is being used.